### PR TITLE
Minor updates to kustomize set image

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,4 @@ testbin/*
 *.swp
 *.swo
 *~
+.vscode/*
```
Comment (marquiz) [inline .gitignore:26]: nit: how this relates to kustomize?

**Review by marquiz — COMMENTED** [review]
We do a lot of small (independent?) changes in this PR and at a quick glance it's a bit hard to follow what/why. I'd suggest splitting the independent/orthogonal changes into separate commits or at least elaborate a bit in the commit message (what was done and why)

File: .gitignore (hunk 1 above)
Comment (ArangoGutierrez) [reply .gitignore:26]: it doesn't but my vscode decided to become verbose recently, and is outputting files and logs there. if you prefer I can drop that line from this PR

Comment (ArangoGutierrez) [reply .gitignore:26]: going to split every change of this PR into mini tiny commits 
this is a nit PR mostly, cosmetic changes here and there

Comment (ArangoGutierrez) [reply .gitignore:26]: Now every misselaneous change lives on it's own commit

File: config/default/kustomization.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 images:
 - name: kube-rbac-proxy
   newName: gcr.io/kubebuilder/kube-rbac-proxy
-  newTag: v0.5.0
+  newTag: v0.8.0
```
Comment (marquiz) [inline config/default/kustomization.yaml:4]: Sorry to nit: is this intended? Related to prometheus?

Comment (ArangoGutierrez) [reply config/default/kustomization.yaml:4]: moved to a separate commit
